docs/broker.rst

```diff
@@ -49,7 +53,8 @@ This will:
 Retrieving files from the store
 *******************************
 
-To retrive files from the store you can use the ``get_file`` interface
+To retrieve files from the store you can use the ``get_file`` interface.
+Importantly, this will only retrieve the path to the file.
```

Comment:
in the docs it is maybe more helpful to say what you should do.  We could also update the api by changing ``get_file`` to ``get_file_path`` for example.  API breaking changes are obviously bad, but I don't think anyone is using this atm.   If it was written by CJ, he was not always the best at naming functions.....